nms/ReflectionProvider/src/net/ess3/nms/refl/ReflUtil.java

```diff
@@ -252,7 +252,7 @@ private NMSVersion(int major, int minor, int release) {
             this.release = release;
         }
         
-        public boolean isHigherThan(NMSVersion o) {
+        boolean isHigherThan(NMSVersion o) {
```

Comment:
These methods should still be public, in case we need to access the NMSVersion elsewhere for any reason.